app/src/main/java/at/aau/serg/activities/GameScreenActivity.kt

```diff
@@ -334,6 +336,37 @@ class GameScreenActivity : AppCompatActivity() {
             allowedToPlayCard = true
             showToast(this, "You are now!")
         }
+
+        // highlight the current player card
+        val calcNextPlayerPosition = calculatePositionOfPlayer(nextPlayerIdx, myPlayerIndex, playerCount)
+        val calcCurrentPlayerPosition = calculatePositionOfPlayer(nextPlayerIdx-1, myPlayerIndex, playerCount)
+        // need to be on an extra thread with sleep!, because in the first round, the playerCardImageViews are not loaded yet
+        Thread {
```

Comment:
I just tried implementing the highlighting of the first load using a fragmentLifecycleCallback, and it works fine. If it is okay with you, I would prefer this so we can get rid of the sleep (I can commit the changes).

Reply:
Yes, please commit this change